wis2box/metadata/station.py

```diff
@@ -35,6 +36,7 @@
 from wis2box.metadata.oscar import get_station_report, upload_station_metadata
 
 LOGGER = logging.getLogger(__name__)
+API_URL = 'http://wis2box-api:80/oapi'
```

Comment:
Derive instead from `wis2box.env.API_URL`

Reply:
wis2box.env.API_URL is the URL outside of the containers... API_URL here is `http://localhost:8999/oapi` which doesn't resolve from within a container.